Tests/ApodiniTests/RESTInterfaceExporterTests.swift

```diff
@@ -381,4 +381,84 @@ class RESTInterfaceExporterTests: ApodiniTests {
         XCTAssertEqual(restoredHeaders.first(name: .authorization), value)
         XCTAssertEqual(restoredHeaders.first(name: .eTag), "W/\"someTag\"")
     }
+    
+    func testRESTInformation() throws {
+        struct BlobInformationHandler: Handler {
```

Comment:
I guess this should be `(NonBlob)InformationHandler`?

Reply:
Good catch, will be changed to `InformationHandler` 👍